src/tests/ftest/launch.py

```diff
@@ -1163,6 +1167,24 @@ def rename_logs(avocado_logs_dir, test_file):
             "Error renaming {} to {}: {}".format(
                 test_logs_dir, new_test_logs_dir, error))
 
+    xml_file = os.path.join(new_test_logs_dir, "results.xml")
+    try:
+        with open(xml_file) as xml_buffer:
+            xml_data = xml_buffer.read()
+    except OSError as error:
+        print("Error reading {} : {}".format(xml_file, str(error)))
+        return
+
+    test_dir = os.path.split(os.path.dirname(test_file))[-1]
+    org_class = "classname=\""
+    new_class = "{}FTEST_{}.".format(org_class, test_dir)
+    xml_data = re.sub(org_class, new_class, xml_data)
+
+    try:
+        with open(xml_file, "w") as xml_buffer:
+            xml_buffer.write(xml_data)
+    except OSError as error:
+        print("Error writing {}: {}".format(xml_file, str(error)))
```

Comment:
Should this be a print or an error?

Reply:
I think a print is reasonable here.  If we raise an exception here we will stop running any additional tests.  If this error occurs all it means is the test result will appear in "Build Artifacts / Functional / job-<date>-<test_dir>-<test_name_wo_py>" instead of "Build Artifacts / Functional / <test_dir> / <test_name_w_py>".  The key is we will still have the test results.

Eventually when the execution of `launch.py` yields its own test results we'll want something more than a print, but that is out of the scope of this PR.

Reply:
Correction: the write error would not affect the Build Artifacts location, but rather how the test is identified in the test results.  This could be a bit confusing, but we would at least have the test results for any subsequent test run.